test/compatibility/compatibility_test_for_releases.sh

```diff
@@ -540,6 +540,37 @@ upgrade_downgrade()
     done
 }
 
+test_dirty_restart()
+{
+    local src_branch="$1"
+    local dst_branch="$2"
+
+    # Don't worry about changing flags for the dst branch, we're going to restart it from the source
+    # branch's working directory anwyay.
+    local flags="-1q $(bflag $src_branch)"
+    local config_file="../../../../CONFIG_${src_branch}"
+
+    # Run format on the source branch until it aborts.
+    pushd "${src_branch}/build/test/format"
+    local dir="RUNDIR.${src_branch}"
+
+    # Ignore the error resulting from the segfault.
+    set +e
+    ./t ${flags} -c "$config_file" -h "$dir" format.abort=1
+    set -e
+    popd
+
+    # We now have a directory with WT files that resulted from a crash. Run the newer version
+    # against those.
+    pushd "${dst_branch}/build/test/format"
+    local dir="../../../../${src_branch}/build/test/format/RUNDIR.${src_branch}"
+    ./t ${flags} -c "$config_file" -h "$dir"
```

Comment:
In order to reopen the database we need to use `-R ` here.
`./t ${flags} -R -h "$dir" format_abort=0`
- We do not need to provide config again it will use the one from the previous run.
- We do need to turn about format.abort this time

Reply:
Thanks -- addressed in 50cc24b.